Makefile

```diff
@@ -16,8 +16,15 @@ CONTROL_KEYSPACE ?= OTTER
 REPLICATION_FACTOR ?= 3
 CLOUDCAFE ?= $(shell which cafe-runner)
 
+mkfile_dir := $(shell dirname "$(MAKEFILE_LIST)")
+
 .PHONY: targets env-precheck
 
+hooks:
+	cp ${mkfile_dir}/scripts/config_check.py ${mkfile_dir}/.git/hooks
+	echo "#/bin/bash" > ${mkfile_dir}/.git/hooks/pre-commit
+	echo "python .git/hooks/config_check.py" >> ${mkfile_dir}/.git/hooks/pre-commit
```

Comment:
@lvh specifically advised against linking because he didn't want someone to just change `config_check.py` to disable stuff.  Granted, modifying `config_check.py` and then running make hooks is the same, but I guess more steps to break things.